### DISCUSSION TITLE
zh/docs/middlewares

### DISCUSSION THREAD

**Discussion by giscus** [casbin/casbin#1452] [Docs comments]
# zh/docs/middlewares

使用内置函数或指定自定义函数

https://casbin.org/zh/docs/middlewares

**Comment by andeya** [comment]
I am a contributor to the Salvo web framework (rust ) and have developed the middleware package (https://github.com/andeya/salvo-casbin). Could you please transfer my repository to the Casbin organization and include it in this documentation?

Issue: https://github.com/casbin/casbin-rs/issues/345
